index.html

```diff
@@ -212,38 +212,9 @@ <h2>
         actually an interface, is sometimes used instead of the more accurate
         "an object implementing the interface <code>Foo</code>.
       </p>
-      <p>
-        The term <dfn>DOM</dfn> is used to refer to the API set made available
-        to scripts in Web applications, and does not necessarily imply the
-        existence of an actual [=Document=] object or of any other [=Node=]
-        objects as defined in the [[DOM]] specification.
-      </p>
-      <p>
-        A DOM attribute is said to be <dfn>getting</dfn> when its value is
-        being retrieved (such as by author script), and is said to be
-        <dfn>setting</dfn> when a new value is assigned to it.
-      </p>
-      <p>
-        The term <dfn>JavaScript</dfn> is used to refer to ECMA262, rather than
-        the official term ECMAScript, since the term JavaScript is more widely
-        known. [[ECMASCRIPT]]
-      </p>
-      <p>
-        The term <dfn>resource</dfn> is used to refer to elements and any other
-        user-initiated fetches throughout this specification. For example, a
-        resource could originate from XMLHttpRequest objects [[XHR]], HTML
-        elements [[HTML]] such as iframe, img, script, object, embed, and link
-        with the link type of stylesheet, and SVG elements [[SVG11]] such as
-        svg.
-      </p>
       <p>
         The term <dfn>cross-origin</dfn> is used to mean non [=same origin=].
```

Comment:
Is this needed? It seems like something that ought to be defined elsewhere.

Reply:
I agree we can remove it. HTML and Fetch are not specifically defining it, so we probably don't have to